src/podman.scss

```diff
@@ -86,4 +86,39 @@
     .show-only-when-narrow {
         display: none;
     }
+
+    .ct-table {
+        td {
+            &[data-label=Owner], &[data-label=CPU], &[data-label=Memory], &[data-label="Disk space"], &[data-label="Used by"] {
+                white-space: nowrap;
+            }
+
+            &[data-label=Owner], &[data-label=CPU], &[data-label=Memory], &[data-label="Disk space"], &[data-label="Created"] {
+                text-align: right;
+            }
+
+            &[data-label=CPU] {
+                min-width: 6ch;
+            }
+
+            &[data-label=Memory] {
+                min-width: 14ch;
+            }
+        }
+    }
+}
+
+/* PatternFly workarounds */
```

Comment:
Do we have report for this in BZ?

Reply:
Not yet. It was already late when I posted this PR.

I need to port it to our overrides file in Cockpit and open a PR. I intend on doing this today.

Reply:
Also, what I _didn't_ mention above is that the `> :empty {}` part is to work around an empty block which we should just not show in the JSX. I'll just port over the gap & margin change (not the `:empty` one... as we should probably remove it in the CSS and fix it in the JSX) to PF and suggest altering it upstream by moving the margin to gap instead of overriding the margin (which is what we have to do in a workaround, of course).

Reply:
Additionally, Cockpit Podman should pull in the PF workarounds from Cockpit as part of the library sync.

Reply:
I found that there's an issue about gap in flex. It's not a 100% match, as it wants to do more. But fixing the gap to use the margin is an obvious first step and definitely part of the issue.

https://github.com/patternfly/patternfly/issues/4523

Reply:
Added in Cockpit @ https://github.com/cockpit-project/cockpit/pull/17043

Reply:
> Also, what I didn't mention above is that the > :empty {} part is to work around an empty block which we should just not show in the JSX

Can you please point me to html where this happens and I can take a look why we render empty elements there

Reply:
It's the line that has a highlight. It's in the details view:

![Screenshot from 2022-02-24 16-31-41](https://user-images.githubusercontent.com/10246/155555345-ee972967-1976-4324-9e74-be9e973840bb.png)

The first div has the id, image, and command. The third div has the created and state. The middle div probably should have content in some cases, but doesn't here.

Reply:
@garrett I think you are running some older version.
I dont see here `<div>` but `<dl>` and empty ones are hidden.

![Screenshot from 2022-02-25 12-24-07](https://user-images.githubusercontent.com/12330670/155707660-86300997-50eb-445b-8425-a4dae3763ae3.png)


Can you please make sure you are fully rebased and building from a clean tree and that your checkout is actually being used? I tried to find where this was updated and I think it is https://github.com/cockpit-project/cockpit-podman/commit/bf1de36cc554c826295a4d777064d859b675ee92 but that is half a year old so not sure.